provider/resource.go

```diff
@@ -2046,6 +2049,7 @@ func Provider() tfbridge.ProviderInfo {
 					},
 				},
 			},
+			"azurerm_batch_application":                 {Tok: azureDataSource(azureBackup, "getApplication")},
```

Comment:
Looks like a wrong module?
```suggestion
			"azurerm_batch_application":                 {Tok: azureDataSource(azureBatch, "getApplication")},
```